src/config.rs

```diff
@@ -425,7 +428,9 @@ mod tests {
             [app]
             [app.playbooks]
               a = "/path/to/playbook.yml"
-        "#.to_string()).unwrap();
+        "#
+                                        .to_string())
+                    .unwrap();
```

Comment:
There's a fix for these ones just landed - https://github.com/nrc/rustfmt/commit/2d267b16d1f28e7ab14cd1cc9953ae21fe1856c6 - maybe worth rebuilding and re-running